elements/bulbs-video/components/revealed.js

```diff
@@ -16,7 +17,6 @@ global.BULBS_ELEMENTS_COMSCORE_ID = '6036328';
 
 let prefixCount = 0;
 function makeGaPrefix () {
-  // ga demands tracker names be alphanumeric
   return `videoplayer${prefixCount++}`;
```

Comment:
The GA prefix doesn't have any impact on tracking @MelizzaP.  It's just a way to namespace the tracker on the page.